gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Dec 10 11:19:41 EST 2014
+#Mon Apr 25 00:16:55 EDT 2016
```

Comment:
Could you remove the modification to this file from this commit?

I believe these are from Android Studio 2.0 - I'd like to bump them in a separate commit.